app/views/articles/new.html.erb

```diff
@@ -32,6 +32,12 @@
             createFormattingToolBar();
           </script> 
         </p>
+      
+        <p class="form-p">
+          <%= form.check_box :location, {}, 1, nil %>
+          <%= form.label :location, 'Add Current Location to Article', class: "checkbox", style: "display: inline;" %>
```

Comment:
a little strange that you're using the `:location` attribute to pass in "I want this thing to be geolocated". Muddies up the nature of the `location` attribute a bit. Another option could be to set `attr_accessor :should_geolocate` on the model (which would make `should_geolocate` callable on an Article instance, but not persist the value to the db) and use that here instead. No strong opinions here, so whatever you think is cleanest works for me.